src/lib/Microsoft.Fx.Portability/Analyzer/DotNetFrameworkFilter.cs

```diff
@@ -36,15 +36,14 @@ public class DotNetFrameworkFilter : IDependencyFilter
             "Windows."
         };
 
-        public bool IsFrameworkAssembly(string name, PublicKeyToken publicKeyToken)
-        {
-            return IsKnownPublicKeyToken(publicKeyToken) || IsKnownName(name);
-        }
+        public virtual bool IsFrameworkMember(string name, PublicKeyToken publicKeyToken)
+            => IsFrameworkAssembly(name, publicKeyToken);
```

Comment:
I feel that I missed something obvious. :) Looks like IsFrameworkMember directly calls into IsFrameworkAssembly. With that, why do we add the new IsFrameworkMember method?

Reply:
This is mostly for testing, but it allows for distinguishing whether it's a framework member or if it's an assembly. There are a number of tests that had a distinction here, so having a separate method (which is overridden in the tests) allows for the same test behavior. In the product code, at the moment they are the same thing.